src/amo/components/CollectionManager/index.js

```diff
@@ -55,6 +55,7 @@ type State = {|
   description?: string | null,
   name?: string | null,
   slug?: string | null,
+  customSlug?: boolean | false,
```

Comment:
I think you don't need `| false` here because we only describe a type, and `boolean` is the type.